source/geod24/concurrency.d

```diff
@@ -332,12 +333,17 @@ public:
      * that a Tid executed in the future will have the same toString() output
      * as another Tid that has already terminated.
      */
-    void toString(scope void delegate(const(char)[]) sink)
+    void toString (scope void delegate(const(char)[]) sink)
```

Comment:
Any style fix, such as this, to `std.concurrency` should be in their own commit. Optimally, it would be great to have:
- First commit: Add `std.concurrency` without any alteration
- Second commit: Stylistic fixes
- Third (and more commit): Necessary changes to `std.concurrency`, and, if needed, LocalRest
- Last commit: Adapt LocalRest to the new concurrency primitive.

What is very important is that there should be minimal changes in the tests. Some element of the API may changes, but the functionalities need to stay there.